core/src/test/java/com/linecorp/armeria/server/http/HttpServerPathTest.java

```diff
@@ -47,15 +52,98 @@ protected void doGet(ServiceRequestContext ctx, HttpRequest req, HttpResponseWri
                 res.respond(HttpStatus.OK);
             }
         });
+
+        sb.service(PathMapping.ofPrefix("/"), new AbstractHttpService() {
+            @Override
+            protected void doGet(ServiceRequestContext ctx, HttpRequest req, HttpResponseWriter res)
+                    throws Exception {
+                res.respond(HttpStatus.OK);
+            }
+        });
+    }
+
+    // last space is workaround..
+    private static final String HTTP11_PROTOCOL = "HTTP/1.1 ";
```

Comment:
This constants used once.  so, This code will be moved into assertion code, like below.
startsWith("HTTP/1.1 " + expected.toString());